{{cookiecutter.directory_name}}/pyproject.toml

```diff
@@ -10,6 +72,18 @@ branch = true
 source = ["{{ cookiecutter.package_name }}"]
 command_line = "-m pytest"
 
+[tool.isort]
+lines_after_imports = 2
+force_single_line = 1
+no_lines_before = ["FUTURE", "STDLIB", "THIRDPARTY", "FIRSTPARTY", "LOCALFOLDER"]
+known_first_party = "{{ cookiecutter.package_name }}"
+src_paths = ["{{ cookiecutter.package_name }}", "tests"]
+line_length = 120
+
+[tool.setuptools]
+include-package-data = true
+packages = ["{{ cookiecutter.package_name }}"]
```

Comment:
After reading up on setuptools [package discovery](https://setuptools.pypa.io/en/latest/userguide/package_discovery.html) and [data file inclusion](https://setuptools.pypa.io/en/latest/userguide/datafiles.html#include-package-data) (check especially the comment in the pyproject.toml example) I think this whole section could be removed. For sure `include-package-data = true` can be removed, because it is now true by default. The package discovery could be removed if we move to a `src`-based layout.

This actually has two advantages:

1. This above, i.e. simpler configuration.
2. It also automatically supports single-file packages (you don't need a folder for a package if it only consists of one file, but to support that you normally need to reconfigure, see e.g. how I did that in [kwandl](https://github.com/egpbos/kwandl/blob/d6909330c0573f22814d2d4a0b1bb8572b77b854/setup.cfg#L36C1-L36C11); I think that with a `src` dir, though, that will also just be autodiscovered, yay!).

```suggestion
```

To be clear, this suggestion would have to be accompanied by adding a `src` dir.

Reply:
I also agree that moving to a src-based layout would be the nicer way to go :)